src/app/static/src/js/components/runWorkflow/runWorkflowTable.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+    <table class="table-bordered">
+        <tr>
+            <th class="p-2">
+                Report
+            </th>
+            <th v-if="anyParams" class="p-2" id="table-params-header">
+                Parameters
+            </th>
+            <th class="p-2">
+                Status
+            </th>
+            <th class="p-2">
+                Logs
+            </th>
+        </tr>
+        <tr v-for="(report, index) in workflowRunStatus.reports" :key="report.key">
```

Comment:
Sorry for not spotting this before. We should be using the sort order of the workflow summary here, not the run status.

The workflow summary doesn't neccessarily respect the ordering of reports provided; it returns reports in a topological sort according to the dependency tree, with reports of the same name grouped together. The reports should be displayed in that order, which corresponds to the order in which they will be run.

This also means that the ordering here is still not consistent, see:

![Screenshot from 2022-03-21 14-39-55](https://user-images.githubusercontent.com/6736820/159288268-d00f1cdb-eb3c-4353-ac1f-2f555ee2e696.png)

And I think your previous comment was correct - there's currently no way to match up run statuses to the workflow summary reports. Looks like we'll have to implement a change somewhere, just trying to work out who is responsible for the ordering of the status response. I think perhaps the thing to do will be to add an `execution_order` column to the `ORDERLYWEB_WORKFLOW_RUN_REPORTS` table and update that when the workflow summary is returned with the topological ordering.

Reply:
Working on this now: https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-3033

Reply:
Very sorry, my bad, Had forgotten that workflow summary did execution ordering too.

Reply:
Yes, these should all be consistently ordered now :)